modules/rostests/apitests/ntdll/NtAllocateVirtualMemory.c

```diff
@@ -674,17 +724,20 @@ CheckSomeDefaultAddresses(VOID)
                                      &Size,
                                      MEM_RESERVE,
                                      PAGE_READWRITE);
+    // Result varies between Windows versions and sometimes between configurations/runs.
```

Comment:
If it varies between different runs, maybe you should find out why and adjust the test to behave consistently.
Or at least add a comment why it fails and handle both gracefully (although it may then be questionable whether the test actually tests anything).